Wrong module

1v opened the pull request:
Causes an error:

```
     NameError:
       uninitialized constant Torckapi::InvalidSchemeError
```

1v committed:
Wrong module

1v committed:
some tests

1v committed:
travis config

1v committed:
adding rake

1v committed:
gemspec

1v committed:
gemspec

1v committed:
some changes

1v committed:
rakefile added

1v committed:
remove Gemfile.lock

1v committed:
codeclimate-test-reporter

1v committed:
env tocken codeclimate

1v committed:
codeclimate-test-reporter fix

1v committed:
implement actual timeout and tries for HTTP

1v committed:
guardfile added

1v committed:
low down number of tries

1v committed:
formatting

1v committed:
cut off strange timeouts; refactor retries

1v committed:
Merge branch 'master' of https://github.com/1v/torckapi

1v committed:
fixtures refactoring

1v committed:
adding tests and fixtures

1v committed:
tests added; fixtures filenames changed

1v committed:
trying to fix 1.9.3 version error

1v committed:
adding parentheses according to https://github.com/bbatsov/ruby-style-guide#method-parens

1v committed:
1.9.3 again; using determination of ruby version

1v committed:
messages for exceptions

krupenik commented:
that's a whole lot of changes for that kind of error. Care to split into logical pieces?

1v commented:
I have no idea how to do that.

1v committed:
Refactoring

1v committed:
Adding license

1v committed:
New badges

1v committed:
Adding latest stable ruby

1v committed:
Codeclimate fix for ruby 1.9.3

1v committed:
Rake version

1v committed:
Codeclimate fixed

1v committed:
Update spec_helper.rb

1v committed:
Typo

1v committed:
Update torckapi.gemspec

1v committed:
Update torckapi.gemspec

1v committed:
Update torckapi.gemspec

1v committed:
Update torckapi.gemspec

1v committed:
Update torckapi.gemspec

1v committed:
Update torckapi.gemspec

1v committed:
json

1v committed:
Update .travis.yml

CaptainJet committed:
Eliminate using ruby's Timeout module

Timeout is noted to be completely unsafe, and unreliable to use.
Originally, this would hang on failed UDP reads forever (noted running
in jruby).
Use IO#select instead, with appropriate timeout.

CaptainJet committed:
Allow proper loading of failed Bencoded data

When returned with a failure of a response, originally this will raise a
MalformedResponseError well, erroneously. The response is proper, and
should carry on to raise the next error later, for proper debugging
purposes.

CaptainJet committed:
Return response directly

Also fix identation, maybe travis tests?

CaptainJet committed:
Revert "Allow proper loading of failed Bencoded data"

This reverts commit b000225b6a3068eb858ed8290fe3594ed849e383.

1v committed:
Fix specs

CaptainJet committed:
Merge pull request #1 from 1v/udp-fix

Fix specs

CaptainJet committed:
Revert "Revert "Allow proper loading of failed Bencoded data""

This reverts commit 5262935bf43adf18d10be95c926df5d643f8b5d5.

1v committed:
Merge pull request #1 from CaptainJet/master

Fix up some bad code practice, and infinite hanging on UDP requests.
